test/e2e/priority_class_test.go

```diff
@@ -45,4 +50,14 @@ func TestToEnsureUserPriorityClassIsPresentAndLower(t *testing.T) {
 	if userPriorityClass.Value >= systemClusterPriorityClass.Value || userPriorityClass.Value >= systemNodePriorityClass.Value {
 		t.Fatalf("openshift-user-critical was higher priority than existing classes: %d", userPriorityClass.Value)
 	}
+	// Ensure openshift-system-critical is a lower value than the system class' ones, but higher than openshift-user-critical.
+	if openshiftSystemPriorityClass.Value >= systemClusterPriorityClass.Value {
```

Comment:
This perhaps is not needed. We any way won't be able to create a priority class greater than system-cluster critical or node critical

Reply:
someone can revise it later.